src/commands/lambda/__tests__/functions/instrument.test.ts

```diff
@@ -99,7 +99,7 @@ describe('instrument', () => {
             "Variables": Object {
               "DD_FLUSH_TO_LOG": "false",
               "DD_LAMBDA_HANDLER": "handler.hello",
-              "DD_MERGE_XRAY_TRACES": "false",
```

Comment:
Why the changes to these tests?

Reply:
I reverted a change and forgot to revert the tests too. Will change them back, thank you for pointing it out.